eks/cluster-autoscaler/cluster_autoscaler.go

```diff
@@ -0,0 +1,19 @@
+package cluster_autoscaler
+
+import (
+	"github.com/aws/aws-k8s-tester/eksconfig"
+	k8s_client "github.com/aws/aws-k8s-tester/pkg/k8s-client"
+	"github.com/aws/aws-sdk-go/service/cloudformation/cloudformationiface"
+	"go.uber.org/zap"
```

Comment:
Let's place standard packages (e.g. `"os"` at top, and 3rd party packages bottom, and run `./scripts/fmt.sh`)